assets/js/components/common/PacketGraph.jsx

```diff
@@ -66,28 +66,28 @@ class PacketGraph extends Component {
       scales: {
         y: {
           title: {
-            text: 'RSSI',
-            display: true
+            text: "RSSI",
+            display: true,
           },
           min: -120,
           max: 20,
         },
         x: {
           title: {
-            text: 'Time Past in Seconds',
-            display: true
+            text: "Time Passed in Seconds",
```

Comment:
main change in this file, rest is linting.

Reply:
lgtm